src/Service/S3/tests/Integration/S3ClientTest.php

```diff
@@ -213,6 +214,18 @@ public function testCreateMultipartUpload(): void
         self::assertNotNull($result->getUploadId());
     }
 
+    public function testDeleteBucket(): void
+    {
+        $client = $this->getClient();
+
+        $input = new DeleteBucketRequest([
+            'Bucket' => 'my_bucket',
+        ]);
+        $result = $client->DeleteBucket($input);
```

Comment:
Could we assert that bucket does not exists aymore? (I don't know if it's implemented by the fakeS3)

Reply:
Sure. I've fixed that.